chemclipse/plugins/org.eclipse.chemclipse.msd.classifier.supplier.molpeak/src/org/eclipse/chemclipse/msd/classifier/supplier/molpeak/classifier/BasePeakClassifier.java

```diff
@@ -103,6 +103,8 @@ public static ILigninRatios calculateLigninRatios(IChromatogramSelection<?, ?> c
 			double count = countArea ? peak.getIntegratedArea() : 1;
 			exitloop:
 			for(IIdentificationTarget peakTarget : peak.getTargets()) {
+				if(!peakTarget.getIdentifier().equals(BasePeakIdentifier.IDENTIFIER))
```

Comment:
Looks good. I recommend to use square brackets if(...){} also for simple if statements. Please also update the year in the header and set yourself as a contributor.

Reply:
Updated.